src/views/SelectProduct.vue

```diff
@@ -0,0 +1,172 @@
+              <ion-item lines="none">
+                <ion-label>{{ $t("Select all variants") }}</ion-label>
+                <ion-toggle />
+              </ion-item>
+            </div>
+          </section>
+
+          <section class="section-grid">
+            <ion-card>
+              <Image src="https://cdn.shopify.com/s/files/1/0069/7384/9727/products/test-track.jpg?v=1626255137" />
+              <ion-item lines="none">
+                <ion-label>
+                  SKU
+                  <p>Color: Blue</p>
+                  <p>Size: XL</p>
+                </ion-label>
+              </ion-item>
+            </ion-card>
+            <ion-card>
+              <Image src="https://cdn.shopify.com/s/files/1/0069/7384/9727/products/test-track.jpg?v=1626255137" />
+              <ion-item lines="none">
+                <ion-label>
+                  SKU
+                  <p>Color: Blue</p>
+                  <p>Size: XL</p>
+                </ion-label>
+              </ion-item>
+            </ion-card>
+          </section>
+          <hr />
+        </main>
+      </div>
+
+      <div class="action desktop-only">
+        <ion-button>
+          {{ $t("Select locations") }}
+          <ion-icon :icon="arrowForwardOutline" />
+        </ion-button>
+      </div>
+
+      <ion-fab vertical="bottom" horizontal="end" slot="fixed" class="mobile-only">
+        <ion-fab-button>
+          <ion-icon :icon="arrowForwardOutline" />
+        </ion-fab-button>
+      </ion-fab>
+    </ion-content>
+  </ion-page>
+</template>
+
+<script lang="ts">
+import Image from '@/components/Image.vue';
+import {
+  IonButton,
+  IonButtons,
+  IonCard,
+  IonContent,
+  IonFab,
+  IonFabButton,
+  IonHeader,
+  IonIcon,
+  IonItem,
+  IonLabel,
+  IonMenuButton,
+  IonPage,
+  IonSearchbar,
+  IonTitle,
+  IonToggle,
+  IonToolbar
+} from '@ionic/vue';
+import { defineComponent } from 'vue';
+import { arrowForwardOutline, downloadOutline, filterOutline } from 'ionicons/icons';
+import { useRouter } from 'vue-router';
+
+export default defineComponent({
+  name: 'SelectProduct',
+  components: {
+    IonButton,
+    IonButtons,
+    IonCard,
+    IonContent,
+    IonFab,
+    IonFabButton,
+    IonHeader,
+    IonIcon,
+    IonItem,
+    IonLabel,
+    IonMenuButton,
+    IonPage,
+    IonSearchbar,
+    IonTitle,
+    IonToggle,
+    IonToolbar,
+    Image
+  },
+  setup() {
+    const router = useRouter();
+
+    return {
+      arrowForwardOutline,
+      downloadOutline,
+      filterOutline,
+      router
+    };
+  },
+});
+</script>
+
+<style scoped>
+.filters {
+  border-right: 1px solid var(--ion-color-medium);
+}
+.section-grid {
+  grid-template-columns: repeat(auto-fill, 200px);
+}
+
+@media (min-width: 991px) {
+  .action {
+    /* desktop-only class is setting display to unset and below properties were not getting applied that is why display is set block here */
+    display: block;
+    /* TODO use spacer variable */
+    margin: 250px 0;
```

Comment:
instead of doing this we should add a position of fixed here and position it at the bottom center

Reply:
Done